app/src/main/java/org/hackillinois/android/view/home/eventlist/EventClickListener.kt

```diff
@@ -1,7 +1,8 @@
 package org.hackillinois.android.view.home.eventlist
 
-import org.hackillinois.android.database.entity.Event
+// import org.hackillinois.android.database.entity.Event
```

Comment:
Delete?

Reply:
I think it's a good idea to leave this, because it is a useful feature for future apps